### DISCUSSION TITLE
Error al utilizar Spinner

### DISCUSSION THREAD

**Discussion by sebalogue** [TiendaNube/nimbus-design-system#72] [Q&A] [answered]
Hola!
Estoy teniendo unos error con nimbus al querer usar el componente Spinner. Quería consultar si alguien sabe si puede ser un error del paquete de nimbus. O si a alguien le pasó algo parecido, porque no estoy encontrando cuál es el error realmente.
Al usar el Spinner en el código:
```
  return (
    <Stack align="stretch" column>
      <Spinner />
...
```

En la consola salta este error:
![image](https://user-images.githubusercontent.com/28818104/211367273-9cd29c53-42aa-48fc-9283-7db11faa1d34.png)

Que al parecer es un error dentro del código del Spinner:
![image](https://user-images.githubusercontent.com/28818104/211367215-9f200b28-1cb4-4028-8240-7a506be0cbb7.png)

**Comment by juanchigallego** [comment, marked as answer]
Hola Seba, ¿cómo estás?

No me quedó muy claro cuál es el error que estás teniendo.
Es con el Spinner de Nimbus v1 o v2? Qué versión estás usando del paquete?

En mi caso, estuve [probando el componente](https://codesandbox.io/s/prod-water-0gvl2u?file=/src/App.tsx) y no parece tener ningún problema.

Si me podés compartir más info del problema podemos intentar recrearlo así lo solucionamos. Gracias!

**Reply by sebalogue** [reply]
Hola Juanchi! todo bien y vos?

Era el de Nimbus v2. 
Ahora ví que agregando `@nimbus-ds/styles` como está en el código que me pasaste funciono perfecto!

Para aclarar, lo que pasaba era que no se mostraba nada en pantalla al intentar renderizar el Spinner y aparecía en la consola el error que pasé, pero con lo de styles se arregló.

Gracias! 😄

**Reply by juanchigallego** [reply]
Genial! Sí, con la última actualización, separamos los estilos de los componentes para mejorar la performance y evitar algunos problemas de redundancia de estilos que estábamos teniendo, con lo cual ahora es necesario importar la hoja de estilos dentro del proyecto separado del componente de React.

Cualquier otra duda avisame!
